app/objects/c_operation.py

```diff
@@ -405,7 +423,10 @@ def fact_to_dict(f):
         await services.get('rest_svc').persist_source(dict(access=[self.access]), data)
 
     async def update_operation(self, services):
-        self.agents = await services.get('rest_svc').construct_agents_for_group(self.group)
```

Comment:
I think we can keep this the way it originally was, no? The rest service method will handle finding the agents based on the group value. This way we're not duplicating code